File: Programming EF Code First/chapter4/Chapter 4 VB/Fluent/Model/Person.vb
Namespace Model
  Public Class Person
	Public Sub New()
	  Address = New Address()
	  Info = New PersonalInfo With {.Weight = New Measurement(), .Height = New Measurement()}
	End Sub

	Public Property PersonId() As Integer
	Public Property SocialSecurityNumber() As Integer
	Public Property FirstName() As String
	Public Property LastName() As String
	Public Property Address() As Address
	Public Property Info() As PersonalInfo

	Public Property PrimaryContactFor() As List(Of Lodging)
	Public Property SecondaryContactFor() As List(Of Lodging)
	Public Property Photo() As PersonPhoto
  End Class
End Namespace

File: Programming EF Code First/chapter4/Chapter4/BAGA_FluentAPI/Model/Person.cs
using System.Collections.Generic;

namespace Model
{
  public class Person
  {
    public Person()
    {
      Address = new Address();
      Info = new PersonalInfo
      {
        Weight = new Measurement(),
        Height = new Measurement()
      };
    }

    public int PersonId { get; set; }
    public int SocialSecurityNumber { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public Address Address { get; set; }
    public PersonalInfo Info { get; set; }

    public List<Lodging> PrimaryContactFor { get; set; }
    public List<Lodging> SecondaryContactFor { get; set; }
    public PersonPhoto Photo { get; set; }
  }
}
